lib/optimizely/optimizely_config.rb

```diff
@@ -152,6 +152,10 @@ def get_features_map(all_experiments_map)
           feature['key'] => {
             'id' => feature['id'],
             'key' => feature['key'],
+            # This experimentsMap is for experiments of legacy projects only.
+            # For flag projects, experiment keys are not guaranteed to be unique
+            # across multiple flags, so this map may not include all experiments
+            # when keys conflict. Use experimentRules and deliveryRules instead.
```

Comment:
This message should be chanaged to: 
"This experimentsMap is deprecated. Use experimentRules and deliveryRules instead."

The one at top level ("experimentsMap" in OptimizleyConfig) looks good.